File: Python/백준/Bronze/25812. Nice Raises/Nice Raises.py
a,b = map(int,input().split())
cnt1 = 0 #고정 인상 급여
cnt2 = 0 #두 배 급여
for i in range(a):
    c = int(input())
    if b>c:
        cnt1+=1
    elif b<c:
        cnt2+=1
if cnt1==cnt2:
    print(0)
elif cnt1>cnt2:
    print(1)
else:
    print(2)

File: C#/백준/Bronze/25812. Nice Raises/Nice Raises.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int a = aa[0]; int b = aa[1]; int cnt1 = 0; int cnt2 = 0;
            for (int i=0;i<a;i++){
                int c = int.Parse(Console.ReadLine());
                if (b>c)
                    cnt1++;
                else if(c>b)
                    cnt2++;
            }
            if (cnt1>cnt2)
                Console.WriteLine(1);
            else if (cnt2>cnt1)
                Console.WriteLine(2);
            else
                Console.WriteLine(0);
        }
    }
}